Resources/doc/reference/configuration/user-context.rst

```diff
@@ -120,6 +120,16 @@ request. However, when you decide to cache hash responses, you must invalidate
 them when the user context changes, particularly when the user logs in or out.
 This bundle provides a logout handler that takes care of this for you.
 
+.. _always_vary_on_context_hash:
+
+``always_vary_on_context_hash``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**type**: ``boolean`` **default**: ``true``
+
+Set ``always_vary_on_context_hash`` to  ``false`` to disable always adding X-User-Context-Hash or Cookie, Authorization in Vary header.
```

Comment:
oh, ok. does that make sense? i think we should change the code then so that it still adds the vary for the lookup request. or do you see a use case where you would not want to vary that?